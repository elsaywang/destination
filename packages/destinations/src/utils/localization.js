import { defineMessages } from 'react-intl';

export const getMessages = defaultMessage =>
    defineMessages({
        title: {
            id: 'components.Layout.title',
            description: `Title for the ${defaultMessage} page`,
            defaultMessage,
        },
    });
