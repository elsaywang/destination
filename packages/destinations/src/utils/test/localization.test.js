import { getMessages } from '../localization';

describe('localization util', () => {
    it('matches snapshot', () => {
        expect(getMessages('Configuration')).toMatchSnapshot();
        expect(getMessages('Destinations')).toMatchSnapshot();
    });
});
