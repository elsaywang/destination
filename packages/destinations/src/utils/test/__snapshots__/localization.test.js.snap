// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`localization util matches snapshot 1`] = `
Object {
  "title": Object {
    "defaultMessage": "Configuration",
    "description": "Title for the Configuration page",
    "id": "components.Layout.title",
  },
}
`;

exports[`localization util matches snapshot 2`] = `
Object {
  "title": Object {
    "defaultMessage": "Destinations",
    "description": "Title for the Destinations page",
    "id": "components.Layout.title",
  },
}
`;
